pkg/errno/errno.go

```diff
@@ -40,7 +40,11 @@ var (
 	UserAlreadyExistErr    = NewErrNo(UserAlreadyExistErrCode, "User already exists")
 	AuthorizationFailedErr = NewErrNo(AuthorizationFailedErrCode, "Authorization failed")
 	BadRequest             = NewErrNo(BadRequestErrCode, "Request Failed")
```

Comment:
need to remove residual traces of merge, such as "<<<<<<<< HEAD"

unmodified documents do not need to be submitted